controllers/ingress/eventhandlers/ingress_events.go

```diff
@@ -52,6 +58,7 @@ func (h *enqueueRequestsForIngressEvent) Delete(e event.DeleteEvent, queue workq
 }
 
 func (h *enqueueRequestsForIngressEvent) Generic(e event.GenericEvent, queue workqueue.RateLimitingInterface) {
+	h.enqueueIfBelongsToGroup(queue, e.Object.(*networking.Ingress))
```

Comment:
Any reason why we are handling this event?

Reply:
the service eventHandler will enqueue a generic Ingress event if there is a OIDC secret change happens